test/Element/CsrfTest.php

```diff
@@ -83,4 +83,18 @@ public function testSetOptionsTraversable(): void
         $this->assertEquals(777, $validator->getTimeOut());
         $this->assertEquals('MySalt', $validator->getSalt());
     }
+
+    public function testNameOverride(): void
+    {
+        $element = new CsrfElement('foo');
+        $element->setOptions([
+            'csrf_options' => [
+                'name' => 'bar',
+            ],
+        ]);
```

Comment:
Is there a reason why the name for the validator should be different from the name for the form element?

Reply:
In my case I'm going around our application making our CSRF tokens unique from eachother, but I don't want to touch the views or other usages of the elements.

Most of the form elements are named "security" meaning they all share the same tokens. Having to look for every case of `->get('security')` and matching it to the correct form and renaming it is going to be a tedious and error-prone endeavor.

Reply:
@froschdesign Any update on this?